da/blob/types.go

```diff
@@ -55,18 +55,18 @@ type BeaconBlockBody interface {
 	Length() uint64
 }
 
-type BeaconBlockHeader[BeaconBlockHeaderT any] interface {
+type BeaconBlockHeader interface {
```

Comment:
@fridrik01 We should be able to remove this interface and just use the concrete type.

Reply:
done!